recipes/photospline/build.sh

```diff
@@ -0,0 +1,8 @@
+
+export CFITSIOROOT=$PREFIX
+export METISROOT=$PREFIX
+export SUITESPARSEROOT=$PREFIX
+export TBB_INSTALL_DIR=$PREFIX
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_INSTALL_LIBDIR=lib .
+make
+make install
```

Comment:
Is there a `make check` or `ctest` we can run here?

PS: try to encapsulate all the env vars like: `${PREFIX}`.